scholarly/data_types.py

```diff
@@ -163,7 +163,7 @@ class Publication(TypedDict, total=False):
     gsrank: int
     author_id: List[str]
     num_citations: int
-    cites_id: int
+    cites_id: List[str]
```

Comment:
Despite the type hinting saying `int`, `cites_id` was of type `str` to begin with. See https://github.com/scholarly-python-package/scholarly/blob/9fe14de6f54ee6eef117ae2ea2743e8ebfe1d13b/scholarly/publication_parser.py#L322